src/Components/Hero/index.tsx

```diff
@@ -9,14 +9,15 @@ export default class Hero extends React.Component {
         return (
             <div className="hero">
                 <div className="hero--info">
-                    <h1>Python Pizza Night</h1>
+                    <h1>Python Pizza</h1>
                     <h2>
-                        <a target="_blank" rel="noopener noreferrer">
-                            Berlin @ Wooga
+                        Berlin @ Wooga <br />
+                        <a href="https://goo.gl/maps/mkQP66yAnXGHdK9w6" target="_blank">
+                            Saarbrücker Str. 38, 10405 Berlin
                         </a>
                     </h2>
-                    <h2>23 August 2019</h2>
-                    <h2>17:00 - 00:30</h2>
+                    <h2>Saturday 16 May 2020</h2>
+                    <h2>10:00 - 17:30 - Doors open at 09:30</h2>
```

Comment:
I agree with Tania that we could remove the "Doors open..." part here to make the headline less dense in text.

Reply:
Wops, sorry, saving issue. Fixed